solr/config/schema.xml

```diff
@@ -361,16 +367,85 @@
 
     <!-- Analyzed Text, general case -->
     <fieldtype name="text" class="solr.TextField" positionIncrementGap="10000" autoGeneratePhraseQueries="true">
-      <analyzer>
+      <analyzer type="index">
+        <tokenizer class="solr.WhitespaceTokenizerFactory" />
+        <filter class="solr.ICUFoldingFilterFactory"/>  <!-- NFKC, case folding, diacritics removed -->
+        <filter class="solr.SynonymFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="false"/>
+        <filter class="solr.SnowballPorterFilterFactory"/>
```

Comment:
One of the big differences between our configurations and the ContentSearch configurations is we were already using the `SnowballPorterFilterFactory` for our full-text fields.  Removing this in our configurations seems to make the existing full-text search (used be Feigenbaum) to return vastly different highlighting (and potentially search results).